blog/docs/articles/performance-test-with-slos.md

```diff
@@ -0,0 +1,183 @@
+
+## Overview of Iter8
+
+Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that Kubernetes apps and ML models perform well and maximize business value.
+
+Iter8 supports the following use-cases.
+
+1.  Performance testing and SLO validation of HTTP services.
+2.  Performance testing and SLO validation of gRPC services.
+3.  SLO validation using custom metrics from any database(s) or REST API(s).
+
+### Iter8 experiment
+Iter8 introduces the notion of an [experiment](https://iter8.tools/0.11/getting-started/concepts/#iter8-experiment), which is a list of configurable tasks that are executed in a specific sequence.
+
+<img src="/blog/docs/articles/images/iter8-intro-dark.png" alt="Iter8 experiment" width="600"/>
+
+Iter8 packs a number of powerful features that facilitate Kubernetes app testing and experimentation. They include the following.
+
+1.  **Generating load and collecting built-in metrics for HTTP and gRPC services.** Simplifies performance testing by eliminating the need to setup and use metrics databases.
+2.  **Well-defined notion of service-level objectives (SLOs).** Makes it simple to define and verify SLOs in experiments.
+3.  **Custom metrics.** Enables the use of custom metrics from any database(s) or REST API(s) in experiments.
+4.  **Readiness check.** The performance testing portion of the experiment begins only after the service is ready.
+5.  **HTML/text reports.** Promotes human understanding of experiment results through visual insights.
+6.  **Assertions.** Verifies whether the target app satisfies the specified SLOs or not after an experiment. Simplifies automation in CI/CD/GitOps pipelines: branch off into different paths depending upon whether the assertions are true or false.
+7.  **Multi-loop experiments.** Experiment tasks can be executed periodically (multi-loop) instead of just once (single-loop). This enables Iter8 to refresh metric values and perform SLO validation using the latest metric values during each loop.
+8.  **Experiment anywhere.** Iter8 experiments can be launched inside a Kubernetes cluster, in local environments, or inside a GitHub Actions pipeline.
+
+## Quick start
+
+Install the Iter8 CLI using `brew` as follows. You can also install using pre-built binaries as described [here](https://iter8.tools/0.11/getting-started/install/).
+
+```shell
+brew tap iter8-tools/iter8
+brew install iter8@0.11
+```
+
+### Tutorial: Performance test for Knative HTTP service
+In this tutorial, we will launch an Iter8 experiment that generates load for a Knative HTTP service, collects Iter8's built-in HTTP metrics, and validates the specified service-level objectives (SLOs). This experiment is illustrated in the figure below.
+
+<img src="/blog/docs/articles/images/iter8-http.png" alt="HTTP performance test" width="800"/>
```

Comment:
This image does not load properly...

<img width="478" alt="image" src="https://user-images.githubusercontent.com/44698588/179822578-8643b02b-d307-4fde-a1bd-6101bbe347d5.png">

Reply:
Hopefully, this is now fixed.

Reply:
`<img src="../images/iter8-intro-dark.png" alt="Iter8 experiment" width="600"/>` should work

Reply:
`<img src="../images/iter8-grpc.png" alt="gRPC performance test" width="800"/>`

Reply:
Fixed.